README.md

```diff
@@ -89,23 +90,26 @@ The following table shows how the properties from the Unified Catalog object are
 
 ### Subscription handling
 
-Unified Catalog subscription and Marketplace are different.
-For Unified Catalog there is one subscription per application per Catalog Item
-From the Marketplace it will be translated to one subscription per plan and per application to guarantee that the provider can enforce the quota correct, that is defined in the product plan.
+Unified Catalog subscriptions and Marketplace subscriptions are different.
+
+For the Unified Catalog, there is one subscription per application per Catalog Item.
+
+From the Marketplace this will be translated into one subscription per plan and application to guarantee that the provider can correctly enforce the quota that is defined in the product plan.
 
 ![Alt text](subscription.png)
 
-Due to that, the migration script creates only 1 subscription for each individual resources available in the product. The subscription is reused to add the Access to the various application.
```

Comment:
The migration script will create only 1 subscription that can be used with each resource in the product. This subscription is then used to add access to the variuos applications.